<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoAnalysisClass</class>
 <widget class="QMainWindow" name="VideoAnalysisClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoAnalysis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDial" name="output_dial">
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="notchTarget">
       <double>3.700000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="screen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="open_video_action"/>
    <addaction name="detect_action"/>
    <addaction name="exit_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>快捷栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="open_video_action">
   <property name="text">
    <string>打开(&amp;O)...</string>
   </property>
  </action>
  <action name="detect_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>检测(&amp;D)</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>退出 (&amp;X)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="video_analysis.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit_action</sender>
   <signal>triggered()</signal>
   <receiver>VideoAnalysisClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
